static/bidder-info/tpmn.yaml

```diff
@@ -0,0 +1,20 @@
+endpoint: "https://gat.tpmn.io/ortb/pbs_bidder"
+maintainer:
+  email: "prebid@tpmn.io"
```

Comment:
@tpmn-admin , Prebid team has sent verification email to confirm accuracy of above email. Requesting to respond back on email with "received" message

Reply:
received

Reply:
@tpmn-admin  please respond back on email

Reply:
@onkarvhanumante sorry for late. I thought I would reply to the mail, but it didn't get sent. I replied to your email a few minutes ago.